lighthouse-extension/gulpfile.js

```diff
@@ -174,6 +194,7 @@ gulp.task('compilejs', () => {
       file.contents = new Buffer(minified);
       return file;
     }))
+    .pipe(banner(BANNER))
```

Comment:
Sure, I can take a dependency, just didn't want to introduce another one but yeah dev dependency doesn't really matter